tests/core/contracts/test_contract_panic_errors.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import re
+
+from tests.core.contracts.utils import (
+    deploy,
+)
+from web3._utils.contract_error_handling import (
+    PANIC_ERROR_CODES,
+)
+from web3._utils.contract_sources.contract_data.panic_errors_contract import (
+    PANIC_ERRORS_CONTRACT_DATA,
+)
+from web3.exceptions import (
+    ContractPanicError,
+)
+
+
+@pytest.fixture
+def panic_errors_contract(w3, address_conversion_func):
+    panic_errors_contract_factory = w3.eth.contract(**PANIC_ERRORS_CONTRACT_DATA)
+    return deploy(w3, panic_errors_contract_factory, address_conversion_func)
+
+
+@pytest.mark.parametrize(
+    "panic_error,params",
+    (
+        ("01", []),
+        ("11", []),
+        ("12", [0]),
```

Comment:
This is kind of annoying but I had to pass these invalid values in, rather than write them into the contract, since Solidity checks these before compilation and doesn't allow it.